### PR TITLE
helm mode: Add hubble enable / disable commands

### PR DESCRIPTION
**Pull request by michi-covalent**
3 commits:

- implement hubble enable / disable commands
- add GKE test coverage for helm mode
- temporary commit to test GKE workflow change. to be removed before merging.

### CODE REVIEW

**Review by aanm — APPROVED** [review]

File: internal/helm/helm.go [hunk 1]
```diff
@@ -438,3 +441,31 @@ func resolveChartVersion(versionFlag string) (semver2.Version, *chart.Chart, err
 	}
 	return version, helmChart, nil
 }
+
+// UpgradeCurrentRelease upgrades the existing Helm release with given Helm values.
+func UpgradeCurrentRelease(
+	ctx context.Context,
+	namespace string,
+	helmValues map[string]interface{},
+	resetValues bool,
```
Comment (chancez) [inline internal/helm/helm.go:450]: nit: Having too boolean arguments next to each other is likely to cause issues down the line, perhaps consider using a struct for some of these arguments, especially anything that's not required (namespace, helmValues, resetValues, reuseValues)

**Review by chancez — APPROVED** [review]
Overall LGTM.

File: internal/helm/helm.go (hunk 1 above)
Comment (michi-covalent) [reply internal/helm/helm.go:450]: yeah makes sense, i'll add a struct for helm-related parameters 👍

**Review by sayboras — APPROVED** [review]
LGTM ✅, the GKE workflow is good now, so I reckon you can remove the temp commit.

**Review by nathanjsweet — APPROVED** [review]

**Comment by michi-covalent** [comment]
thank you for your review all! i'll remove the DO NOT MERGE commit and merge this 🚀

**Pull request merged**